rucio-server/values.yaml

```diff
@@ -311,14 +311,6 @@ config:
     ## config.webui.usercert:  (default "/opt/rucio/etc/usercert_with_key.pem")
     # usercert: "/opt/rucio/etc/usercert_with_key.pem"
 
-resources: {}
```

Comment:
Please keep this here and just change to `serverResources`. Additionally, add `authServerResources`. It's personal preference but I like when you can get possible configuration values directly from the default values.yaml which you can just copy over.